users/pcoves/unicode.c

```diff
@@ -0,0 +1,20 @@
+#include "unicode.h"
+#include "pcoves.h"
+
+bool process_record_unicode(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case EMOTE0:
+            if (record->event.pressed) send_unicode_hex_string("0028 256f 00B0 25A1 00B0 FF09 256F FE35 253B 2501 253B");
```

Comment:
You can use `send_unicode_string()` here instead to not have to convert these to hex :)